ipyleaflet/leaflet.py

```diff
@@ -846,6 +855,8 @@ class Map(DOMWidget, InteractMixin):
     default_style = InstanceDict(MapStyle).tag(sync=True, **widget_serialization)
     dragging_style = InstanceDict(MapStyle).tag(sync=True, **widget_serialization)
 
+    zoom_control = Bool(True).tag(sync=True)
```

Comment:
No need to have `.tag(sync=True)`, this value will be used only Python side and won't be synchronized with the JavaScript side anymore.